lib/smart_proxy_ansible/roles_reader.rb

```diff
@@ -33,6 +30,10 @@ def logger
 
         private
 
+        def config_file_path
+          File.join ::Proxy::Ansible::Plugin.settings.ansible_working_dir, '.ansible.cfg'
```

Comment:
this is wrong. In the working_directory we should look for `/ansible.cfg` according to https://docs.ansible.com/ansible/latest/reference_appendices/config.html
And shouldn't we acrually implement a fallback to the default? :thinking:

The fact that working dir is by default == to home dir should not IMO matter, user can change that and get confused or we can decide to change the default in the future.
I think we should go with `<working_directory>/ansible.cfg` with fallback to `~<rex/ansible user>/.ansible.cfg` and maybe even fallback to the system config.
Do you think we can do something like that ? :thinking: